Makefile

```diff
@@ -45,7 +45,7 @@ BUILDCMD = go build -o $(TARGET) $(VERBOSE_FLAG) -ldflags "-X github.com/heptio/
 BUILD = $(BUILDCMD) $(GOTARGET)/cmd/sonobuoy
```

Comment:
should we add a build target here too?

Reply:
I don't think so since we don't have anything to build for this lib